internal/infra/azuredevops/queue.go

```
@@ -259,3 +272,167 @@
+		return fmt.Errorf("failed to create request: %w", err)
+	}
+
+	req.Header.Set("Authorization", buildAuthHeader(a.PATToken))
+	req.Header.Set("Content-Type", "application/json")
+
+	client := &http.Client{Timeout: 10 * time.Second}
+	resp, err := client.Do(req)
+	if err != nil {
+		return fmt.Errorf("failed to execute request: %w", err)
+	}
+	defer resp.Body.Close()
```

golangci-lint (1.64.8)

Error return value of `resp.Body.Close` is not checked

(errcheck)